pages/common/crunch.md

```diff
@@ -0,0 +1,36 @@
+# crunch
+
+> Wordlist generator.
+> More information: <https://sourceforge.net/projects/crunch-wordlist/>.
+
+- Output a list of words of length 1 to 3 with only lowercase characters:
+
+`crunch {{1}} {{3}}`
+
+- Output a list of hexadecimal words of length 8:
+
+`crunch {{8}} {{8}} {{0123456789abcdef}}`
+
+- Output a list of all permutations of abc (lengths are not processed):
+
+`crunch {{1}} {{1}} -p {{abc}}`
+
+- Output a list of all permutations of the given groups (lengths are not processed):
+
+`crunch {{1}} {{1}} -p {{abc}} {{def}} {{ghi}}`
+
+- Output a list of words generated according to the given pattern and setting the maximum duplicate number of letters:
+
+`crunch {{5}} {{5}} {{abcde123}} -t {{@@@}}`
```

Comment:
The example is invalid. I get this output:
```
The maximum and minimum length should be the same size as the pattern you specified. 
min = 5  max = 5  strlen(@@@)=3
```

Reply:
@navarroaxel Thanks for the heads-up ! I will correct and re-test the command.

Reply:
@navarroaxel Corrected.